docs/docs/integrations/embedding-stores/tablestore.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 16
+---
+
+# Tablestore
+
+https://www.aliyun.com/product/ots
+
+## Maven Dependency
+
+```xml
+
+<dependency>
+    <groupId>dev.langchain4j</groupId>
+    <artifactId>langchain4j-tablestore</artifactId>
+    <version>{latest-version}</version>
```

Comment:
```suggestion
    <version>0.35.0</version>
```

Reply:
done.    I also upgraded the version in pom from 0.34.0-SNAPSHOT to 0.35.0-SNAPSHOT.